pkgs/by-name/ap/appflowy/package.nix

```
@@ -30,7 +30,7 @@ let
 in
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "appflowy";
-  version = "0.7.1";
+  version = "0.7.4";
```

Update to latest version
```suggestion
  version = "0.7.7";
```